src/setupProxy.js

```diff
@@ -7,11 +7,13 @@
   and changes the URL of the requests to use the real backend.
 */
 
+import { cleanUrl } from "./utils/utils";
+
 // Note that this currently works only as a CommonJS module
 const { createProxyMiddleware } = require("http-proxy-middleware");
 
-const clientUrl = process.env.REACT_APP_CLIENT_URL;
-const basicAuth = process.env.REACT_APP_BASIC_AUTH || null;
+const CLIENT_URL = new URL(cleanUrl(process.env.REACT_APP_CLIENT_URL));
+const BASIC_AUTH = process.env.REACT_APP_BASIC_AUTH === undefined ? null : new URL(cleanUrl(process.env.REACT_APP_BASIC_AUTH), CLIENT_URL)
```

Comment:
BASIC_AUTH is a string with credentials, not a URL

Reply:
Fixed!